conda/environments/all_cuda-129_arch-x86_64.yaml

```diff
@@ -25,7 +25,7 @@ dependencies:
 - dask-cuda==25.08.*
 - dask-cudf==25.08.*
 - gcc_linux-64=11.*
-- legate==25.11.*,>=0.0.0.dev0
+- legate==25.11.00.rc1,>=0.0.0.dev0
```

Comment:
This change shouldn't have been necessary, only the other changes should have been necessary and any new commit to retrigger CI.

(Not that it matters much, rc is preferred over dev versions anyway.)